src/ResourceManager/AnalysisServices/Commands.AnalysisServices.ServiceManagement/Microsoft.Azure.Commands.AnalysisServices.ServiceManagement.dll-help.xml

```diff
@@ -0,0 +1,181 @@
+﻿<?xml version="1.0" encoding="utf-8"?><helpItems schema="maml">
```

Comment:
For new cmdlets, please use the PlatyPS help system, as described here: https://github.com/Azure/azure-powershell/blob/dev/documentation/platyPSHelp-documentation.md